Assets/Scripts/Dialogue/InkManager.cs

```diff
@@ -346,7 +353,29 @@ private void DialogueVariablesSetup()
         {
             story.BindExternalFunction("ProgressionEvent", () => { ProgressionEvent(); });
         }
+
+        story.BindExternalFunction("BossDamage", (int pDamage) => { BossDamage(pDamage); });
```

Comment:
I was gonna say you should only bind the external function when the dialogue the player is engaging in is with a boss, but after trying it out it doesn't seem to cause any problems if you do so when there's no BossDamage() in the ink file. I guess you could still do only bind it against bosses for performance reasons or something in that case, but it doesn't seem to cause any problems if you don't

Reply:
you should definitely unbind the function after though (check line 333 for how) since I think it does cause problems for the manager if you leave them bound